test-requirements.txt

```diff
@@ -0,0 +1 @@
+ansible-lint==2.3.6
```

Comment:
Only ansible-lint `2.3.6` is working after there will be sudo/become errors that are not merged yet.